dependencies.yaml

```diff
@@ -814,7 +814,7 @@ dependencies:
               - pandas==2.0.*
           - matrix: {dependencies: "latest"}
             packages:
-              - numba-cuda==0.0.15
+              - numba-cuda==0.2.0
```

Comment:
This is backwards. The "latest" configuration should use `*numba-cuda-dep`, and the "oldest" configuration should use `numba-cuda==0.2.0`.

Reply:
Fixed in https://github.com/rapidsai/cudf/pull/17628/commits/617ad0c03d1f6f396b84cdf615f144760a3910a1